README.rst

```diff
@@ -12,12 +8,65 @@ FURY
         :target: https://codecov.io/gh/fury-gl/fury
 
 
-Free Unified Rendering in Python
+FURY - Free Unified Rendering in Python
+=======================================
+
+
+FURY is a package for scientific visualization with Python.
+
+- **Website and Documentation:** https://fury.gl
+- **Mailing list:** https://mail.python.org/mailman3/lists/fury.python.org
+- **Official source code repo:** https://github.com/fury-gl/fury.git
+- **Download releases:** https://pypi.org/project/fury/
+- **Issue tracker:** https://github.com/fury-gl/fury/issues
+- **Free software:** 3-clause BSD license
+
+Installation
+============
+
+
+1. from binaries
+~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+    pip install fury
+
+2. from source
+~~~~~~~~~~~~~~
+
+**Step 1.** Get the latest source by cloning this repo::
+
+    git clone https://github.com/fury-gl/fury.git
+
+**Step 2.** Install requirements::
+
+    pip install -r requirements/default.txt
+
+**Step 3.** Install fury via::
+
+    pip install .
+
+or::
+
+    pip install -e .
+
+**Step 4:** Enjoy!
+
+For more information, see also `installation page on fury.gl <https://fury.gl/stable/installation.html>`_
+
+Contribute
+==========
+
+
+We love contributions!
+
+Whatever your experience level, all new contributors are welcomed!
```

Comment:
Remove this entire sentence. Just add something saying:
Start your pull request here (and here add the link to the pull requests of fury).